splash.yaml

```
@@ -0,0 +1,14 @@
+  color_dark: "#272727"
+  image: "assets/images/splash.png"
+  image_dark: "assets/images/splash.png"
+
+  android_12:
+    color: "#ffffff"
+    color_dark: "#272727"
+    image: "assets/images/splash.png"
+    image_dark: "assets/images/splash.png"
+
+  web: false
+  
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)